src/ui/hooks/useOnlineStatusEffect.ts

```diff
@@ -2,14 +2,12 @@ import { useEffect } from "react";
 import { useAppSelector } from "../../store/hooks";
 import { getIsOnline } from "../../store/reducers/stateCache";
 
-const useOnlineStatusEffect = (
-  callback: (onlineStatus: boolean) => void | never
-) => {
+const useOnlineStatusEffect = (callback: () => void | never) => {
   const isOnline = useAppSelector(getIsOnline);
 
   useEffect(() => {
     if (!isOnline) return;
-    callback(isOnline);
+    callback();
```

Comment:
this parameter was never used I think. @Sotatek-DukeVu correct me if I'm wrong

Reply:
Yes. You can remove it. I passed `isOnline` to callback function because I thought maybe they need it. But we can get it from redux.